### PR TITLE
fix(videoButton): Added disabling button while call is active

### LINKED ISSUE #4004: Incorrect display and behaviour of call buttons
##### Brief Description

OS: Arch Linux  
qTox version: master 
Commit hash:  af711c0527048a64ee74e58979217e49d4f39a64
toxcore:  0.1.1
Qt:  5.7.1 

Reproducible: Always

##### Observed Behavior

A regression introduced as part of 917812da has caused incorrect appearance of call related buttons in the chat form, as seen below:

Compressed buttons and incorrect call button colors

Not only are the mute call and mute microphone buttons compessed, when initiating an audio call, the video call button remains green and clickable but non-functional.

##### Expected Behavior

Screenshot of a very similar call taken from the v1.6.0 tag:

Correct button apperance

##### Additional Info

Compressed buttons seem to be fixable by reverting the removal of hard-coded heights and widths in the css files for buttons, though the incorrect video call button may require more in depth analysis to fix.

I am using a HIDPI display.

### PR DESCRIPTION
**Pull request by sudden6**
Fix #4004

Cherry-picked and slighly modified from #3935

---
This change is [Reviewable](https://reviewable.io/reviews/qtox/qtox/4006)

### CODE REVIEW

File: src/widget/form/chatform.cpp [hunk 1]
```diff
@@ -465,34 +469,34 @@ void ChatForm::onVideoCallTriggered()
 void ChatForm::updateCallButtons()
 {
     CoreAV* av = Core::getInstance()->getAv();
-    if (av->isCallActive(f))
+    bool audio = av->isCallActive(f);
+    bool video = av->isCallVideoEnabled(f);
+    callButton->setEnabled(audio && !video);
+    videoButton->setEnabled(video);
+    if (audio || video)
```
Comment (Diadlo) [inline src/widget/form/chatform.cpp]: if `video => audio`, than you can leave only `audio`

Comment (Diadlo) [reply src/widget/form/chatform.cpp]: `=>` is implication

Comment (sudden6) [reply src/widget/form/chatform.cpp]: @Diadlo I don't understand, please explain again.

Comment (sudden6) [reply src/widget/form/chatform.cpp]: ah :) thx

**Comment by Diadlo** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/4006#-:-KZqEUN5EXVWEpvvzKF7:bnfp4nl)*

**Pull request merged**